cirq-core/cirq/experiments/readout_confusion_matrix.py

```diff
@@ -26,12 +26,26 @@
     import cirq
 
 
+def _mitigate_single_bitstring(z_rinv_all: list[np.ndarray], bitstring: np.ndarray) -> float:
+    """Return the mitigated Pauli expectation value for a single observed bitstring.
+
+    Args:
+        z_rinv_all: A list of single-qubit pauli-Z (as a vector) contracted with the single-qubit
+            inverse response matrix, for each qubit.
+        bitstring: The measured bitstring.
+
+    Returns:
+        The corrected expectation value of ZZZZZ... given the single measured bitstring.
+    """
+    return np.prod([z_rinv[bit] for z_rinv, bit in zip(z_rinv_all, bitstring)])
+
+
 class TensoredConfusionMatrices:
     """Store and use confusion matrices for readout error mitigation on sets of qubits.
 
     The confusion matrix (CM) for one qubit is:
 
-        [ Pr(0|0) Pr(1|0) ]
+        [ Pr(0|0) Pr(0|1) ]
```

Comment:
was the old documentation wrong or is this a result of the changes?

Reply:
The old documentation was wrong.